Formula/ktor.rb

```diff
@@ -0,0 +1,28 @@
+class Ktor < Formula
+  desc "CLI tool that creates porjects with ktor framework setup"
+  homepage "https://github.com/ktorio/ktor-cli"
+  url "https://github.com/ktorio/ktor-cli/archive/refs/tags/0.0.1-tmp.tar.gz"
+  sha256 "bdb311910a3afcebe3428aee6fd1ef8c19ac670d089f362a6fb89030c3edf873"
+  license "Apache-2.0"
+
+  depends_on "gradle" => :build
+
+  uses_from_macos "curl"
+  uses_from_macos "unzip"
+
+  def install
+    osArch = Hardware::CPU.arm? ? "Arm" : "X"
+    osName = OS.mac? ? "macos#{osArch}64" : "linuxX64"
```

Comment:
Is there a arm Linux version? Homebrew doesn't actively support arm on Linux but it seems we can add it here without too much effort.

Reply:
Kotlin/Native doesn't have a target for Linux/Arch as far as I know.

Small update: I have to remove macosArm64 target from here because not all of the dependencies are present in gradle for this target. macosX64 binary will suffice to be working for M1 in this case.

Reply:
I mean, if you run `gradle linkMacosArm64` it will fail due to some critical Kotlin/Native dependencies are not present for Macos Arm64, and these dependencies are required for Ktor so I cannot go without them

Reply:
But rosetta is `x86_64` so you should add `depends_on arch: :x86_64` to indicate it does not work on ARM.

Reply:
Thank you! Will do so